### PR TITLE
Add whoami to bioblend.galaxy.config

### PR DESCRIPTION
**Pull request by cat-bro**
Add method to get /api/whoami and basic tests for bioblend.galaxy.config functions

### CODE REVIEW

File: bioblend/galaxy/config/__init__.py [hunk 1]
```diff
@@ -48,3 +48,21 @@ def get_version(self):
...
+    def whoami(self):
+        """
+        Return information about the current authenticated user.
+
+        :rtype: dict
+        :return: Information about current authenticated user
+          For example::
+            {'active': True,
+             'deleted': False,
+             'email': 'jgillard83649163@student.unimelb.edu.au',
```
Comment (simonbray) [inline bioblend/galaxy/config/__init__.py:61]: :laughing:

**Review by simonbray — APPROVED** [review]

File: bioblend/galaxy/config/__init__.py (hunk 1 above)
Comment (nsoranzo) [reply bioblend/galaxy/config/__init__.py:61]:
```suggestion
             'email': 'user@example.org',
```

File: bioblend/galaxy/config/__init__.py [hunk 2]
```diff
@@ -48,3 +48,21 @@ def get_version(self):
...
+             'id': '4aaaaa85aacc9caa',
+             'last_password_change': '2021-07-29T05:34:54.632345',
+             'model_class': 'User',
+             'username': 'julia'}
+        """
+        url = '/'.join((self.gi.url, 'whoami'))
```
Comment (nsoranzo) [inline bioblend/galaxy/config/__init__.py:67]:
```suggestion
        url = self.gi.url + "/whoami"
```

**Comment by cat-bro** [comment]
Thanks @simonbray  and @nsoranzo , suggestions committed

**Review by nsoranzo — APPROVED** [review]

**Comment by nsoranzo** [comment]
Thanks @cat-bro !

**Pull request merged**